testing/loki_alerts/Couchbase-Server/CB90070-permissionDenied/memcached.log.000000.txt

```diff
@@ -0,0 +1,2 @@
+│2022-02-09T14:13:17.631996+00:00 WARNING (travel-sample) CouchKVStore::openDB: error:error opening file [Permission denied]
```

Comment:
Is the `|` prefix part of the log files, or just an artifact of how you looked at them (lnav maybe)? If so, you'll probably want to remove it, as the samples should be raw, barely-modified real log lines.

Reply:
Ah sorry I missed that, fixed now :)